src/Symfony/Component/Security/Core/Authorization/AccessDecisionManager.php

```diff
@@ -177,27 +176,4 @@ private function decideUnanimous(TokenInterface $token, array $attributes, $obje
 
         return $this->allowIfAllAbstainDecisions;
     }
-
-    /**
-     * TokenInterface vote proxy method.
-     *
-     * Acts as a BC layer when the VoterInterface is not implemented on the voter.
-     *
-     * @deprecated as of 3.4 and will be removed in 4.0. Call the voter directly as the instance will always be a VoterInterface
-     */
-    private function vote($voter, TokenInterface $token, $subject, $attributes)
```

Comment:
we could now update the docblock of the constructor so that the type hint for the `$voters` argument is now `VoterInterface[]` instead of `iterable|VoterInterface[]`